ste/mgr-JobPartMgr.go

```diff
@@ -583,8 +583,34 @@ func (jpm *jobPartMgr) resourceDstData(fullFilePath string, dataFileToXfer []byt
 	return common.ResourceHTTPHeaders{ContentType: jpm.inferContentType(fullFilePath, dataFileToXfer), ContentLanguage: jpm.httpHeaders.ContentLanguage, ContentDisposition: jpm.httpHeaders.ContentDisposition, ContentEncoding: jpm.httpHeaders.ContentEncoding, CacheControl: jpm.httpHeaders.CacheControl}, jpm.metadata
 }
 
+// TODO do we want these charset=utf-8?
+var builtinTypes = map[string]string{
+	".css":  "text/css; charset=utf-8",
+	".gif":  "image/gif",
+	".htm":  "text/html; charset=utf-8",
+	".html": "text/html; charset=utf-8",
+	".jpeg": "image/jpeg",
+	".jpg":  "image/jpeg",
+	".js":   "application/javascript",
+	".mjs":  "application/javascript",
+	".pdf":  "application/pdf",
+	".png":  "image/png",
+	".svg":  "image/svg+xml",
+	".wasm": "application/wasm",
+	".webp": "image/webp",
+	".xml":  "text/xml; charset=utf-8",
+}
+
 func (jpm *jobPartMgr) inferContentType(fullFilePath string, dataFileToXfer []byte) string {
-	if guessedType := mime.TypeByExtension(filepath.Ext(fullFilePath)); guessedType != "" {
+	fileExtension := filepath.Ext(fullFilePath)
+
+	// short-circuit for common static website files
+	// mime.TypeByExtension takes the registry into account, which is most often undesirable in practice
+	if override, ok := builtinTypes[fileExtension]; ok {
```

Comment:
@zezha-msft I just realised, we should do strings.ToLower() on the extension before using it to look up in the map.  Otherwise we'll still be broken if the extension is .HTML or something.